src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Card-Strings.json

```diff
@@ -393,5 +393,9 @@
   "stsjorbsmod:FlameWard": {
     "NAME": "Flame Ward",
     "DESCRIPTION": "The next time you are attacked, apply !M! stsjorbsmod:Burning to ALL attackers and gain !B! Block. Exhaust."
+  },
+  "stsjorbsmod:PurgeMind": {
+    "NAME": "Purge Mind",
+    "DESCRIPTION": "Exhaust all cards which \"remember\". NL Deal !stsjorbsmod:BaseDamage! for each card Exhausted. Exhaust. NL Snap."
```

Comment:
I think this should use `!D!` rather than `!stsjorbsmod:BaseDamage!`, since the damage-per-card is going to be modified by strength/weak/etc

Reply:
Twas a relic of an initial implementation, can change.